src/sonic-yang-mgmt/sonic_yang.py

```diff
@@ -16,79 +16,88 @@ def __init__(self, yang_dir):
         try:
             self.ctx = ly.Context(yang_dir)
         except Exception as e:
-            print(e)
+	    self.fail(e)
+
+    def fail(self, e):
+	print(e)
```

Comment:
Ideally, we should have printed something meaningful, such as "Loading Yang model Failed" or "Validation Failed". Because exception can be due to NoneType. And it is difficult to trace why it failed on a switch. 

As of now, I think, we should go ahead with merge. Thx.